app/controllers/people_controller.rb

```diff
@@ -0,0 +1,22 @@
+class PeopleController < ApplicationController
+  def index
+    @displaypeople = Person.all
+    render :index
+    # This isn't necessary since it's implicit. However
+    # we should prefer clarity.
+  end
+
+  def show
+    @id = params[:id]
+    @person = Person.find(@id)
```

Comment:
Since the `@person` variable has the associated id - i think it would be preferred to only use the one `@person` instance variable and retrieve the ID from that instead of having another instance variable